### PR TITLE
Update TR Locales

### PR DESCRIPTION
**Pull request by mikropsoft**

### CODE REVIEW

File: app/src/main/res/values-tr/strings.xml [hunk 1]
```diff
@@ -1,122 +1,140 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
-    <!-- Bottom navigation -->
+<resources>
```
Comment (Malopieds) [inline app/src/main/res/values-tr/strings.xml]: Here the resources should not be there

**Review by Malopieds — CHANGES_REQUESTED** [review]
Need some changes

File: app/src/main/res/values-tr/strings.xml (hunk 1 above)
Comment (mikropsoft) [reply app/src/main/res/values-tr/strings.xml]: Sorry for mistake!

**Comment by Malopieds** [comment]
Full stack trace:

`/home/runner/work/InnerTune/InnerTune/app/src/main/res/values-tr/strings.xml:12: Error: For locale "tr" (Turkish) the following quantity should also be defined: one (e.g. "Sepetinizde 1 X var. Bunu almak istiyor musunuz?") [MissingQuantity]
    <plurals name="n_selected">
    ^

   Explanation for issues of type "MissingQuantity":
   Different languages have different rules for grammatical agreement with
   quantity. In English, for example, the quantity 1 is a special case. We
   write "1 book", but for any other quantity we'd write "n books". This
   distinction between singular and plural is very common, but other languages
   make finer distinctions.

   This lint check looks at each translation of a <plural> and makes sure that
   all the quantity strings considered by the given language are provided by
   this translation.

   For example, an English translation must provide a string for
   quantity="one". Similarly, a Czech translation must provide a string for
   quantity="few".

   https://developer.android.com/guide/topics/resources/string-resource.html#Plurals`

I see you deleted also missingQuantities, if not important you can but here it's not compiling anymore so you should add it back, or fix the plurals.

**Comment by mikropsoft** [comment]
> Full stack trace:
> 
> `/home/runner/work/InnerTune/InnerTune/app/src/main/res/values-tr/strings.xml:12: Error: For locale "tr" (Turkish) the following quantity should also be defined: one (e.g. "Sepetinizde 1 X var. Bunu almak istiyor musunuz?") [MissingQuantity] ^
> 
> Explanation for issues of type "MissingQuantity": Different languages have different rules for grammatical agreement with quantity. In English, for example, the quantity 1 is a special case. We write "1 book", but for any other quantity we'd write "n books". This distinction between singular and plural is very common, but other languages make finer distinctions.
> 
> This lint check looks at each translation of a and makes sure that all the quantity strings considered by the given language are provided by this translation.
> 
> For example, an English translation must provide a string for quantity="one". Similarly, a Czech translation must provide a string for quantity="few".
> 
> [https://developer.android.com/guide/topics/resources/string-resource.html#Plurals`](https://developer.android.com/guide/topics/resources/string-resource.html#Plurals%60)
> 
> I see you deleted also missingQuantities, if not important you can but here it's not compiling anymore so you should add it back, or fix the plurals.

I didn't actually delete anything, and there's nothing missing. I synchronized everything with the English file, and even the number of lines is the same. Now, I'm reviewing it.

File: app/src/main/res/values-tr/strings.xml [hunk 2]
```diff
@@ -147,124 +167,145 @@
         <item quantity="one">%d çalma listesi</item>
         <item quantity="other">%d çalma listesi</item>
     </plurals>
-    <plurals name="n_week" tools:ignore="MissingQuantity">
```
Comment (Malopieds) [inline app/src/main/res/values-tr/strings.xml]: Here you've deleted the tools:ignore, you should either let the ignore or follow android's standards.

File: app/src/main/res/values-tr/strings.xml [hunk 3]
```diff
@@ -147,124 +167,145 @@
...
+    <plurals name="n_week">
         <item quantity="one">%d hafta</item>
         <item quantity="other">%d hafta</item>
     </plurals>
-    <plurals name="n_month" tools:ignore="MissingQuantity">
```
Comment (Malopieds) [inline app/src/main/res/values-tr/strings.xml]: Same here

File: app/src/main/res/values-tr/strings.xml [hunk 4]
```diff
@@ -147,124 +167,145 @@
...
+    <plurals name="n_month">
         <item quantity="one">%d ay</item>
         <item quantity="other">%d ay</item>
     </plurals>
-    <plurals name="n_year" tools:ignore="MissingQuantity">
```
Comment (Malopieds) [inline app/src/main/res/values-tr/strings.xml]: Same here

**Review by Malopieds — CHANGES_REQUESTED** [review]

**Comment by Malopieds** [comment]
Thank you for this PR  and bearing with me !

**Pull request merged**